src/Services/SignatureFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace LearnositySdk\Services;
+
+use LearnositySdk\Services\Signatures\SignatureInterface;
+use LearnositySdk\Services\Signatures\HashSignature;
+use LearnositySdk\Services\Signatures\HmacSignature;
+
+class SignatureFactory
+{
+    const LEGACY_SIGNATURE_VERSION = '01';
+
+    const DEFAULT_SIGNATURE_VERSION = '02';
```

Comment:
I think we should not redefine those, and directly use the const from the implentations, such as ```HashSignature::SIGNATURE_VERSION```

Reply:
Good shout, done.